src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+function Person({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+      {person.isMarried === true ? (
+        <p className="Person__partner">
+          {`${person.partnerName} is my ${
+            person.sex === 'm' ? 'wife' : 'husband'
```

Comment:
create a separate variable and use it here

`const partner = person.sex === 'm' ? 'wife' : 'husband';`